bindings/rust/bench/src/s2n_tls.rs

```diff
@@ -72,28 +74,67 @@ impl S2NHarness {
         }
     }
 
-    fn create_config(mode: Mode, crypto_config: &CryptoConfig) -> Result<Config, Box<dyn Error>> {
-        let security_policy = match (&crypto_config.cipher_suite, &crypto_config.ec_group) {
+    fn create_common_config_builder(
```

Comment:
```suggestion
    fn shared_config(
```

Reply:
I think having "create" in the name makes it clear there aren't any side effects to the function, and having "builder" in the name makes it clear that you don't get a readily usable config but rather a builder with the common options set.